README.md

```diff
@@ -41,20 +39,24 @@ InfluxDB reporter for [Telemetry](https://github.com/beam-telemetry/telemetry)
 $ make test
 ```
 
-It should setup the latest InfluxDB in docker and run all the tests against it.
+It should setup the latest InfluxDB in docker for v1 and the specified InfluxDB from quay for v2 and run all the tests against them.
```

Comment:
🙋‍♂ v2 is still a Docker container and the fact that it comes from quay is an implementation detail. I think this could be simplified to "It should setup the latest InfluxDB in docker for both v1 and v2 and runs all the tests against them."